setup.cfg

```diff
@@ -98,4 +93,4 @@ console_scripts =
 	talisker.gunicorn.eventlet=talisker:run_gunicorn_eventlet[gunicorn]
 	talisker.gunicorn.gevent=talisker:run_gunicorn_gevent[gunicorn]
 	talisker.celery=talisker:run_celery[celery]
-	talisker.help=talisker:run_help
+	talisker.help=talisker:run_help
```

Comment:
This seems to be an unwanted change. You probably deleted at the empty line at the end of the file.